### PR TITLE
Personal Portfolio: Add Contact section

### PR DESCRIPTION
**Pull request by helio3197**
### Milestone 3:
Added Contact section to the Portfolio project.
#### Added files:
- Social media icons
#### Updated files:
- index.html: Added Contact section with a functional input form for user data.
- styles.css: Added styles to the Contact section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,165 +1,232 @@
...
+        <li>
+          <a href=""><span class="fingers-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="twitter-icon-dark"></span></a>
+        </li>
+        <li>
+          <a href=""><span class="m-icon-dark"></span></a>
+        </li>
+      </ul>
```
Comment (moise10r) [inline index.html:229]: - [x] Add this bar at the end of the icons social media :point_down: 

- Your design:



- The Original :

File: index.html [hunk 2]
```diff
@@ -1,165 +1,232 @@
...
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form-section">
+      <h1 class="contact-description">
```
Comment (moise10r) [inline index.html:184]: - [x] Kindly change the `h1` tag into` h2` .... `h6`, I always the best practice to only have one h1 in your index.html

**Review by moise10r — CHANGES_REQUESTED** [review]
### **Hi @helio3197 ** , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (helio3197) [reply index.html:184]: Hi, according to this from a Microverse lesson http://diveintohtml5.it/semantics.html#:~:text=And%20in%20HTML5%2C%20each%20section%20can%20have%20its%20own%20%3Ch1%3E%20element.
in HTML5 each section can have its own h1 element

**Review by AleksandarIvezic — APPROVED** [review]
## Hi @helio3197 👋🏼 , 
### STATUS: Approved! ✔️ ✔️ ✔️ 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 🥂 

gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**